packages/app/src/embed/components/Actions/index.js

```diff
@@ -1,6 +1,9 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
```

Comment:
@DannyRuchtie @tromika basically this is the template to follow to create new (or update) experiments.